spec/factories/notes.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
```

Comment:
Want to use the faker gem?

Reply:
I don't know if that's really needed here. If I understood it correctly, the test we write for the students should be fairly simple just so that they can see how tests are written. Therefore I don't see the necessity to include the complexity of a faker-generated resource.